frontend/lib/map/map/screen_parent_resizer.dart

```diff
@@ -0,0 +1,58 @@
+import 'dart:math';
+import 'package:flutter/material.dart';
+
+/// A widget which resizes the child widget twice when the orientation changes.
+///
+/// * Firstly, the child is resized to
+///   `width = height = max(screenWidth, screenHeight)`. This creates a square.
+/// * Secondly, the child is resized to
+///   `width = parentWidth, height = parentHeight. This sizes the child according to the parent constraints.
```

Comment:
Are you sure, these two steps are definitely necessary? Is it possible to omit the first step?

Reply:
Sadly it is required